go.spec

```diff
@@ -2,10 +2,12 @@
 %define debug_package %{nil}
 %undefine _missing_build_ids_terminate_build
 
-%define _go_rel 1.17
-%define _go_patch 8
+%define _go_rel 1.18
+%define _go_patch 0
 
 %if (0%{?suse_version} > 0)
+# Sigh. SuSE.
+Epoch:		1
```

Comment:
Did SUSE really bump the Epoch?

Reply:
> No, but they set their `Release:` to `1.8.1`, so the NVR is `go1.18-1.18-1.8.1`. Meaning that our package was never going to "win". In an ideal world, we would just use the distro package, but that gets complicated from a release standpoint, I guess?

Does it?  How so?  I would think if the goal of our rolling up a package is to make sure that we are using the latest and greatest, and if the distro is using one even newer, the distro one is meeting (and exceeding when compared to ourselves) the goal, isn't it?  [Why] do we even need to include distros in this project that are maintaining packages for the latest release?

> It's easiest to just say we're building with the latest binaries put out by the Go team and check the box instead of arguing with people.

But when the distro is even newer, isn't that still checking the box?

Ultimately my concern is messing with the `Epoch`.  I feel like that comes back to haunt at some point.